app/views/partials/footer.ejs

```diff
@@ -55,16 +55,14 @@
       >
     </div>
 
-
     <div
-    class="flex items-center justify-center self-stretch mt-4 mb-8 text-sm lg:text-base text-center"
-  >
-  <img src="/static/images/analytics.svg" alt="Provided by Simple Analtics" class="mr-2" />
-    <a target="_blank" class="font-bold text-cvs-green-dark hover:text-cvs-red hover:underline" href="https://simpleanalytics.com/<%= baseUrl %>"
-    ><%= __('View Website Statistics')%></a
-  >
-</div>
-      
+      class="flex items-center justify-center self-stretch mt-4 mb-8 text-sm lg:text-base text-center"
+    >
+      <img src="/static/images/analytics.svg" alt="Provided by Simple Analtics" class="mr-2" />
+        <a target="_blank" class="font-bold text-cvs-green-dark hover:text-cvs-red hover:underline" href="https://simpleanalytics.com/<%= baseUrl %><% if (!shouldAddAnalytics) { %>?ref=<%= baseUrl %><% } %>"
```

Comment:
If we append `?ref=...` my tool does not record the document referrer, but only the one in the URL param.

Reply:
Could use ternary here for improved readability, but not something you have to change :)